src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/EqualsAvoidNullCheck.java

```diff
@@ -68,6 +68,32 @@
  * <pre>
  * &lt;module name=&quot;EqualsAvoidNull&quot;/&gt;
  * </pre>
+ * <p>
+ * Example when ignoreEqualsIgnoreCase is set to false:
+ * </p>
+ * <pre>
+ * String nullString = null;
```

Comment:
It is more frequent case if method is called on method arg, please change example

Reply:
Can I get some clarification about what needs to be changed? Is it nullString? Is it because in most cases users aren't comparing a String to a nullString? Do you want me to use an example like:

```userInput.equals(someString) // violation if userInput is null  ```

Reply:
No, something like
```
boolean testMethod(String someString) {
  return someString.equals("value"); 
}
```

It is more frequent and not obvious case than explicit null string

Reply:
Thank you very much for the clarification, I'll make the changes.

Reply:
@strkkk Sorry for the delay again, but I'd just like to ask if this is suitable or if I should add a method that is not a violation.  
```
boolean equalsValue(String someString){
    return someString.equals("value"); // violation
}

boolean equalsValueIgnoreCase(String someString){
    return someString.equalsIgnoreCase("value"); // violation
}
```

Reply:
Not sure I understood. Your code example looks ok to me. Correct code with OK comments would be nice to have as well